pkg/cloudprovider/aws/instancetype.go

```diff
@@ -97,12 +97,18 @@ func (i *InstanceType) AWSNeurons() *resource.Quantity {
 	return resources.Quantity(fmt.Sprint(count))
 }
 
-// Overhead calculations copied from
-// https://github.com/awslabs/amazon-eks-ami/blob/5a3df0fdb17e540f8d5a9b405096f32d6b9b0a3f/files/bootstrap.sh#L237
+// Computes overhead for https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable
+// Overhead calculations copied from https://github.com/bottlerocket-os/bottlerocket#kubernetes-settings
 func (i *InstanceType) Overhead() v1.ResourceList {
 	overhead := v1.ResourceList{
-		v1.ResourceCPU:    *resource.NewMilliQuantity(int64(0), resource.DecimalSI),
-		v1.ResourceMemory: *resource.NewMilliQuantity((11*i.Pods().Value())+255, resource.DecimalSI),
+		v1.ResourceCPU: *resource.NewMilliQuantity(
+			100, // system-reserved
```

Comment:
Where did `100` come from? I didn't see it in the bottlerocket link (maybe I just missed it?)

Reply:
https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable